tests/test_pv_forecast.py

```diff
@@ -121,7 +121,7 @@ def test_update_ac_power_measurement(pv_forecast_instance, sample_forecast_start
     forecast_start = pv_forecast_instance.get_forecast_start()
     assert forecast_start == sample_forecast_start
 
-    updated = pv_forecast_instance.update_ac_power_measurement(forecast_start, 1000)
+    updated = pv_forecast_instance.update_ac_power_measurement(1000, forecast_start)
```

Comment:
I don't mind, but is there a reason the order got changed?

Reply:
the updated value is not optional, so had to move to the front.